cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -412,6 +428,15 @@ struct ColumnChunkDesc {
 
   int32_t src_col_index{};   // my input column index
   int32_t src_col_schema{};  // my schema index in the file
+
+  float list_bytes_per_row_est{};  // for LIST columns, an estimate on number of bytes per row
```

Comment:
`float` type?

Reply:
It could be a size_t, but since this is just an estimate I figured leaving it as a float couldn't hurt.